src/FixedDataTableRow.js

```diff
@@ -201,17 +221,29 @@ class FixedDataTableRowImpl extends React.Component {
     var style = {
       width: this.props.width,
       height: this.props.height + subRowHeight,
+      zIndex: this.props.zIndex ? this.props.zIndex : 0,
+      position: 'absolute',
     };
+    if (!this.props.visible) {
+      style.display = 'none';
+    }
+    FixedDataTableTranslateDOMPosition(
+      style,
+      0,
+      this.props.offsetTop || 0,
+      this._initialRender,
+      this.props.isRTL
+    );
+
```

Comment:
FixedDataTableTranslateDOMPosition is used to set the position of the row from the top according to its offsetTop